### PR TITLE
Problem: Missing raw account event projection

### PR DESCRIPTION
**Pull request by vincentysc**

### CODE REVIEW

File: projection/account_raw_event/account_raw_event.go [hunk 1]
```diff
@@ -0,0 +1,530 @@
...
+
+	eventsView := NewAccountRawEvents(rdbTxHandle)
+	totalView := NewAccountRawEventsTotal(rdbTxHandle)
+
+	eventRows := make([]view.AccountRawEventRow, 0)
+
+	for _, event := range events {
+		if rawBlockCreatedEvent, ok := event.(*event_usecase.BlockRawEventCreated); ok {
+			accountRawEvent := utils.NewParsedTxsResultLogEvent(&rawBlockCreatedEvent.Data.Content)
+			if rawBlockCreatedEvent.Data.Type == "coin_spent" {
```
Comment (calvinaco) [inline projection/account_raw_event/account_raw_event.go:110]: This list is so magnificent. I have some follow-up questions in mind:

1. Have you read @davcrypto findings on this before https://github.com/crypto-com/chain-indexing/issues/383 ? I think one caveat of relying on event is that some events are not emitted between versions.
2. I wonder if every message handling / chain logic happens emit some kinds of events. Maybe @vincentysc you can share you insight.

**Review by calvinaco — COMMENTED** [review]
Thanks for the PR. I left some follow-up comments.

File: projection/account_raw_event/account_raw_event.go (hunk 1 above)
Comment (davcrypto) [reply projection/account_raw_event/account_raw_event.go:110]: i think there is no need to switch between `event type`, coz raw event should have no parsing on the event data (other than base64 decode)

Comment (vincentysc) [reply projection/account_raw_event/account_raw_event.go:110]: oh, the switch case is mainly for extracting the related accounts.

Comment (vincentysc) [reply projection/account_raw_event/account_raw_event.go:110]: or is there any better way to extract it?

Comment (davcrypto) [reply projection/account_raw_event/account_raw_event.go:110]: can just assuming the event is related to a specific if there is any event value started with address prefix

Comment (vincentysc) [reply projection/account_raw_event/account_raw_event.go:110]: cool, let me update it then~

File: projection/account_raw_event/account_raw_event.go [hunk 2]
```diff
@@ -0,0 +1,155 @@
...
+	totalView := NewAccountRawEventsTotal(rdbTxHandle)
+
+	eventRows := make([]view.AccountRawEventRow, 0)
+
+	for _, event := range events {
+		if rawBlockCreatedEvent, ok := event.(*event_usecase.BlockRawEventCreated); ok {
+			accounts := []string{}
+			// append related account
+			for _, attribute := range rawBlockCreatedEvent.Data.Content.Attributes {
+				if strings.HasPrefix(attribute.Value, projection.accountAddressPrefix) {
```
Comment (calvinaco) [inline projection/account_raw_event/account_raw_event.go:111]: I suggest to do a Cosmos address validness check too. After all having a attribute value prefixed with account address prefix, e.g. `cro` may not be that uncommon actually.

**Review by davcrypto — APPROVED** [review]

**Pull request merged**